darwin/.env.example

```
@@ -32,3 +32,20 @@ IOS_CERTIFICATE_NAME="Apple Distribution: Your Team Name (TEAM_ID)"
 # iOS Device Installation
 # Device name for installing the app (find with: xcrun devicectl list devices)
 IOS_DEVICE_NAME="Your Device Name"
+# macOS App Store Signing Configuration
+# Copy this file to .env and fill in your values
+
+# Path to macOS provisioning profile from App Store Connect
+MACOS_PROFILE_PATH="$(pwd)/darwin/macos.provisionprofile"
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The MACOS_PROFILE_PATH key is duplicated

(DuplicatedKey)